### PR TITLE
use avoid-shelf-pose instead of move-arm-body->bin to avoid quick move

### LINKED ISSUE #1689: Move too quickly in :move-arm-body->bin
Arms move too quickly so that some heavy objects fall down.
- [x] quick start action

### PR DESCRIPTION
**Pull request by knorth55**
i will test it on real robot.
Closes #1689

### CODE REVIEW

File: jsk_2016_01_baxter_apc/euslisp/main-stow.l [hunk 1]
```diff
@@ -113,7 +113,8 @@
             (progn
               (ros::ros-info "[main] ~a, ~a" arm state)
               (ros::ros-info "[main] ~a, place object in bin ~a" arm target-bin)
-              (send *ri* :move-arm-body->bin arm target-bin :scale 5.0)
+              (send *baxter* :avoid-shelf-pose arm target-bin)
+              (send *ri* :send-av 5000)
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/main-stow.l]: I think it is better to use `angle-vector-sequence` like https://github.com/start-jsk/jsk_apc/blob/master/jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l#L602-609

Comment (knorth55) [inline jsk_2016_01_baxter_apc/euslisp/main-stow.l]: modify to use `:angle-vector`, it is same as `angle-vector-sequence`

**Comment by knorth55** [comment]
tested and it works

File: jsk_2016_01_baxter_apc/euslisp/main-stow.l [hunk 2]
```diff
@@ -113,7 +113,12 @@
             (progn
               (ros::ros-info "[main] ~a, ~a" arm state)
               (ros::ros-info "[main] ~a, place object in bin ~a" arm target-bin)
-              (send *ri* :move-arm-body->bin arm target-bin :scale 5.0)
+              (send *ri* :angle-vector-sequence
+                    (list
+                      (send *baxter* :avoid-shelf-pose arm target-bin)
+                      (send *ri* :ik->bin-entrance arm target-bin
+                            :offset (float-vector -200 0 (gethash :offset-avoid-bin-top hcv))))
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/main-stow.l]: `hcv` is not defined here.

Comment (knorth55) [inline jsk_2016_01_baxter_apc/euslisp/main-stow.l]: fixed
thank you

File: jsk_2016_01_baxter_apc/euslisp/main-stow.l [hunk 3]
```diff
@@ -113,9 +113,14 @@
...
               (ros::ros-info "[main] ~a, place object in bin ~a" arm target-bin)
-              (send *ri* :move-arm-body->bin arm target-bin :scale 5.0)
-              (send *ri* :wait-interpolation)
               (setq hcv (send *ri* :get-val '_hard-coded-variables))
+              (send *ri* :angle-vector-sequence
+                    (list
+                      (send *baxter* :avoid-shelf-pose arm target-bin)
+                      (send *ri* :ik->bin-entrance arm target-bin
+                            :offset (float-vector -200 0 (gethash :offset-avoid-bin-top hcv))))
+                    :fast nil 0 :scale 5.0)
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/main-stow.l]: Indent?

**Comment by pazeshun** [comment]
+1

**Pull request merged**